src/react/components/StyleEditor/Preview/area.js

```diff
@@ -63,7 +64,7 @@ export const getAreaSVG = (areaParams, areaFills) => {
    path.setAttribute('stroke-linejoin', linejoin);
 
    const patternId = 'patternPreview' + patternIdCounter++;
-   const fillPatternSVG = getPatternSVG(areaFills, pattern);
+   const fillPatternSVG = getPatternSVG(pattern);
    fillPatternSVG.setAttribute('stroke', color);
    path.setAttribute('fill', 'url(#' + patternId + ')');
```

Comment:
Does this work properly since the patternId isn't used for the pattern anymore? I don't remember if this was a fix for some non-chrome browser to have the id change or was it so that opening, closing and reopening the form left the pattern unfilled. But if it now works without it I think we can remove it. If it works only because we get lucky with generated patterns matching here and in getPatternSVG() we should do something about it.

Reply:
getPatternSVG returns parsePathFromSVG(patternSVG.data) and path is passed to _composeSvgPattern. So it doesn't matter what patternId getFillOption returns. So this should work but the solution is quite messy. Maybe I will create another PR to simplify svg and pattern creation.